tcms/testruns/templates/testruns/get.html

```diff
@@ -0,0 +1,98 @@
+            <span class="fa fa pficon-topology"></span>{% trans 'Test plan' %}:
+            <a href="{% url 'test_plan_url_short' object.plan.pk %}">TR-{{ object.plan.pk }}: {{ object.plan.name }}</a>
+        </h2>
+
+        <h2 class="card-pf-title" style="text-align: left">
+            <span class="fa fa-shopping-cart"></span>{% trans 'Product' %}:
+            <a href="{% url 'testruns-search' %}?product={{ object.product_version.product_id }}" title="Search test runs of {{ object.product_version.product }}">{{ object.product_version.product }}</a>
+        </h2>
+
+        <h2 class="card-pf-title" style="text-align: left">
+            <span class="fa fa-random"></span>{% trans 'Version' %}:
+            <a href="{% url 'testruns-search' %}?product={{ object.plan.product_id }}&product_version={{ object.plan.product_version_id }}" title="Search test runs of {{ object.plan.product_version.value }}">{{ object.plan.product_version.value }}</a>
+        </h2>
+
+        <h2 class="card-pf-title" style="text-align: left">
+            <span class="fa fa-wrench"></span>{% trans 'Build' %}:
+            <a href="{% url 'testruns-search' %}?product={{ object.plan.product_id }}&product_version={{ object.plan.product_version_id }}&build={{ object.build_id }}" title="Search test runs of {{ object.build_id }}">{{ object.build }}</a>
+        </h2>
+
+        <h2 class="card-pf-title" style="text-align: left">
+            <span class="fa pficon-user"></span>{% trans 'Manager' %}:
+            <a href="{% url 'tcms-profile' object.manager.username %}">{{ object.manager.username }}</a>
+        </h2>
+
+        {% if object.default_tester %}
+            <h2 class="card-pf-title" style="text-align: left">
+                <span class="fa fa-search"></span>{% trans 'Default tester' %}:
+                <a href="{% url 'tcms-profile' object.default_tester.username %}">{{ object.default_tester.username }}</a>
+            </h2>
+        {% endif %}
+
+        <h2 class="card-pf-title" style="text-align: left">
+            {% if object.stop_date %}
+                <span class="fa fa-check-square"></span>{% trans 'Status' %}:
+                <span>{% trans "Finished" %}</span>
+                {% if perms.testruns.change_testrun %}
+                    <button class="btn btn-default set-running-button">{% trans "Set to Running" %}</button>
+                {% endif %}
+            {% else %}
+                <span class="fa fa-play"></span>{% trans 'Status' %}:
+                <span>{% trans "Running" %}</span>
+                {% if perms.testruns.change_testrun %}
+                    <button class="btn btn-default set-finished-button">{% trans "Set to Finished" %}</button>
+                {% endif %}
+            {% endif %}
+        </h2>
+
+        <h2 class="card-pf-title" style="text-align: left">
+            <span class="fa fa-calendar"></span>{% trans 'Started at' %}:
+            {{ object.start_date|date:"Y-m-d H:i:s" }}
```

Comment:
Remove `|date` filter b/c it is inconsistent with how timestamps are shown everywhere else.